src/data/content/assessment/question.ts

```diff
@@ -556,6 +556,61 @@ function removeResponsesWithNoMatch(question: Question): Question {
   });
 }
 
+const isStarMatch = match => match === '*';
+
```

Comment:
Can you add some comments here that describe the logic behind this?  Why this was added, what it is checking for, etc?